howdoi/howdoi.py

```diff
@@ -621,7 +623,16 @@ def howdoi(raw_query):
         cache.set(cache_key, res)
     except (RequestsConnectionError, SSLError):
         res = {'error': f'Unable to reach {args["search_engine"]}. Do you need to use a proxy?\n'}
-
+    except BlockError:
+        BLOCKED_ENGINES.append(args['search_engine'])
+        if BLOCKED_ENGINES == SUPPORTED_SEARCH_ENGINES:
+            return howdoi(args)
+        for eng in SUPPORTED_SEARCH_ENGINES:
+            if eng not in BLOCKED_ENGINES:
+                logging.error('%sRetrying search with %s%s', GREEN, eng, END_FORMAT)
```

Comment:
Let's use `logging.info` here since it's not quite an error yet since we have a fallback we can use.

Reply:
So I tried using `logging.info` too, but `INFO` logs only show up when you use `--explain` which normally the user doesn't go for. If you don't use the `--explain` argument, you see that the search engine blocked your request, but you still got the answer (the rerouting to Google isn't visible as `logging.info` will not show up).  I've left it as `logging.error` but let me know if `logging.info` would still be the preferred route, I'll change it to that